README.md

```diff
@@ -67,14 +67,14 @@ objectql(source, { a: function noop() {} }); // returns {}
 
 
 // valid query key values
-objectql(source, { a: true }); // returns { a: 'a' }
-objectql(source, { b: true }); // returns { b: { c: 'c', d: 'd' } }
-objectql(source, { a: true, b: { c: true } }); // returns { a: 'a', b: { c: 'c' } }
+objectql(source, '{ a }'); // returns { a: 'a' }
```

Comment:
Here's where we should show how to do the same query as an object / string.

Reply:
I think this covers the bases: https://github.com/bmullan91/objectql/pull/1/commits/1a8f088487af11f91615af2c0f2b2fd325e99d84